example/deployment/local-docker/bash-scripts/runDockerExample.sh

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env bash
 
+. ./example/deployment/bash-scripts/setScriptPath.sh
+
 # Check that the required network is present
 NETWORKS=$(docker network ls -f 'name=example_palisade' --format '{{.Name}}')
 if [[ $(grep -c example_palisade <(echo $NETWORKS)) -lt 1 ]]; then
     echo "Docker network \"palisade_example\" is not present. Do you need to run the dockerComposeUp.sh script?"
```

Comment:
Is this network name correct? It differs from the one above it.

Reply:
```suggestion
    echo "Docker network \"example_palisade\" is not present. Do you need to run the dockerComposeUp.sh script?"
```